packages/web-components/examples/codesandbox/components/leaving-ibm/src/index.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import '@carbon/ibmdotcom-web-components/es/components/leaving-ibm/leaving-ibm-modal';
+import '@carbon/ibmdotcom-web-components/es/components/leaving-ibm/leaving-ibm-modal-body';
+import '@carbon/ibmdotcom-web-components/es/components/leaving-ibm/leaving-ibm-modal-heading';
+import 'carbon-web-components/es/components/modal/modal-header';
+import 'carbon-web-components/es/components/modal/modal-close-button';
+import 'carbon-web-components/es/components/button/button';
```

Comment:
Please refer to: #4437

```suggestion
import '@carbon/ibmdotcom-web-components/es/components/leaving-ibm/leaving-ibm-modal.js';
import '@carbon/ibmdotcom-web-components/es/components/leaving-ibm/leaving-ibm-modal-body.js';
import '@carbon/ibmdotcom-web-components/es/components/leaving-ibm/leaving-ibm-modal-heading.js';
import 'carbon-web-components/es/components/modal/modal-header.js';
import 'carbon-web-components/es/components/modal/modal-close-button';
import 'carbon-web-components/es/components/button/button.js';
```